### DISCUSSION TITLE
Help with intro files on icecast please

### DISCUSSION THREAD

**Discussion by Vicinis** [AzuraCast/AzuraCast#5724] [Q&A]
For this scenario, I run 2 icecast stations that I stream from home to my servers offsite. Both stations stream at 320 MP3.
Station A (Main Staion)---> a.example.com
Station B (BackUp Station---> b.example.com
Both stations have intro files also 320 mp3s

Station A has two mounts
/radio.mp3 ---> a.example.com:8000/radio.mp3 with a Fallback Mount /backup
/backup ---> a.example.com:8000/backup.mp3 with a Relay Stream URL that point to  b.example.com:8000/radio.mp3

Station b has one mount
/radio.mp3 ---> b.example.com:8000/radio.mp3

If i go to Station A Azuracast Profile and play /stream.mp3, intro plays first
If i go to Station B Azuracast Profile and play /stream.mp3, intro plays first

If i drop source from Station A, Station B picks up
When i resumes source, Station A resumes.
This is perfect.
What's not working... The intros don't play when switching sources

My questions, how do i get it play intro's when the listeners are changing mounts?

TIA

**Comment by Moonbase59** [comment]
Probably no luck. Icecast plays intro files upon _a new connection_, not on fallbacks.
